### PR TITLE
Add the genesis check for ethbridge module

### PR DESCRIPTION
**Pull request by juniuszhou**

### CODE REVIEW

File: x/ethbridge/genesis_test.go [hunk 1]
```diff
@@ -15,7 +15,6 @@ func TestExportGenesis(t *testing.T) {
 	// Generate State
 	receiver := CreateState(ctx, keeper, t)
 	state := ethbridge.ExportGenesis(ctx, keeper)
-	// assert.Equal(t, len(state.PeggyTokens), tokenscount)
```
Comment (Brando753) [inline x/ethbridge/genesis_test.go:18]: Why was this assertion removed?

File: x/ethbridge/genesis_test.go [hunk 2]
```diff
@@ -25,34 +24,37 @@ func TestInitGenesis(t *testing.T) {
 	// Generate State
 	receiver := CreateState(ctx1, keeper1, t)
 	state := ethbridge.ExportGenesis(ctx1, keeper1)
-	// assert.Equal(t, len(state.PeggyTokens), tokenscount)
```
Comment (Brando753) [inline x/ethbridge/genesis_test.go:28]: same question

File: x/ethbridge/genesis_test.go [hunk 3]
```diff
@@ -25,34 +24,37 @@ func TestInitGenesis(t *testing.T) {
...
 	assert.Equal(t, state.CrosschainFeeReceiveAccount, receiver)
+
+	// not set for state2, so receiver account is empty
 	state2 := ethbridge.ExportGenesis(ctx2, keeper2)
-	// assert.Equal(t, len(state2.PeggyTokens), 0)
```
Comment (Brando753) [inline x/ethbridge/genesis_test.go:31]: same question

File: x/ethbridge/genesis_test.go (hunk 1 above)
Comment (juniuszhou) [reply x/ethbridge/genesis_test.go:18]: no PeggyTokens in state anymore, and assertation already commented.

**Review by Brando753 — APPROVED** [review]

**Pull request merged**